src/main/java/frc/robot/utils/PhoenixUtil.java

```diff
@@ -1,6 +1,4 @@
-// Copyright 2021-2024 FRC 6328
```

Comment:
can you jsut wack a license like this? if you are using their code you probably need to make your stuff just add to the copyright??  i don't know. but when you do things like this you can cause lots of ripples.